test/extensions/dynamic_modules/test_data/rust/no_op.rs

```diff
@@ -0,0 +1,17 @@
+use envoy_proxy_dynamic_modules_rust_sdk::declare_program_init;
+
+declare_program_init!(init);
+
+fn init() -> bool {
+    true
+}
+
+#[no_mangle]
+pub extern "C" fn getSomeVariable() -> i32 {
+    static mut SOME_VARIABLE: i32 = 0;
+
+    unsafe {
+        SOME_VARIABLE += 1;
+        SOME_VARIABLE
+    }
```

Comment:
nit: let's set a good example and not use unsafe for no reason. You can use LazyLock to create an atomic or whatever else you want.

Reply:
I wanted to use LazyLock but somehow only x64 CI was failing after Rust upgrades (LazyLock is stabilized in 1.80 and LazyOnce in 1.79), so I chose to use atomic for now https://github.com/envoyproxy/envoy/pull/35914/commits/5a24889b5dbcb266a241e7a4fd48745587071d28